app/models/chat_room.rb

```diff
@@ -0,0 +1,2 @@
+class ChatRoom < ApplicationRecord
+end
```

Comment:
please add association

Reply:
Added the required associations on the last commit